<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Рейс</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="9">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Авиакомпания</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="3">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Очистить</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3" colspan="2">
    <widget class="QLineEdit" name="lineEdit_aircraft_type"/>
   </item>
   <item row="9" column="4">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>Отменить</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Найти</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3" colspan="2">
    <widget class="QComboBox" name="comboBox_status">
     <item>
      <property name="text">
       <string>Все статусы</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>По расписанию</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Задержан</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Отменен</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Тип ВС</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="QLineEdit" name="lineEdit_airline"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Откуда вылет</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="2">
    <widget class="QLineEdit" name="lineEdit_departure_from"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Пункт назначения</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="2">
    <widget class="QLineEdit" name="lineEdit_destination"/>
   </item>
   <item row="4" column="3" colspan="2">
    <widget class="QDateTimeEdit" name="dateTimeEdit_departure_time_range1">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Прибытие в</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QDateTimeEdit" name="dateTimeEdit_departure_time_range2">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="timeSpec">
      <enum>Qt::LocalTime</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Вылет в</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3" colspan="2">
    <widget class="QDateTimeEdit" name="dateTimeEdit_arrival_time_range1">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QDateTimeEdit" name="dateTimeEdit_arrival_time_range2">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Гейт</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3" colspan="2">
    <widget class="QLineEdit" name="lineEdit_gate"/>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Статус</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="QLineEdit" name="lineEdit_flight"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
